doc/tools/create_changelog.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+from __future__ import absolute_import
+from __future__ import unicode_literals
+from __future__ import print_function
+import sys
+import io
+from subprocess import check_output, CalledProcessError
```

Comment:
Update.  @mbadams5 educated me on the ways of Python and explained that from will limit the code that is brought in thus keeping the script smaller.  He also pointed out that it makes it easier to be future proof with code such as what follows which imports the same functions from different libraries.